RevenueCatUI/CustomerCenter/Views/CompatibilityContentUnavailableView.swift

```diff
@@ -80,6 +78,22 @@ struct CompatibilityContentUnavailableView: View {
         }
 
     }
+    
```

Comment:
this is what the linter is complaining about, just extra spaces within this empty line
```suggestion

```

It's not exactly a big deal 😅 

I'd recommend setting this up in Xcode so you don't keep running into it in the future

<img width="942" alt="Screenshot 2025-01-10 at 4 35 06 PM" src="https://github.com/user-attachments/assets/4fb1542a-f98f-49d0-9738-4edcda6df138" />

cc @ajpallares since you might run into the same thing

Reply:
same in line 88